src/main/resources/com/ncc/aif/restricted_aif.shacl

```diff
@@ -40,6 +40,48 @@ aida:EventArgumentShape
 aida:RelationArgumentShape
     sh:property aida:EdgeJustificationCompound, aida:EdgeJustificationLimit .
 
+#########################
+# CompoundJustification must be used only for justifications of argument assertions,
+# and not for justifications for entities, events, or relation KEs
+#------------------------
+aida:RestrictCompoundJustificationPropertyShape
+   a sh:PropertyShape ;
+   sh:path aida:justifiedBy ;
+   sh:not [sh:class aida:CompoundJustification] ;
+   sh:message "CompoundJustification must be used only for justifications of argument assertions" .
+
+aida:EventShape
+    # may not provide compound justification
```

Comment:
Do you need the same comment for each line?

Reply:
Make as fixed after in person discussion.

Reply:
We agreed to keep it as is for copy-paste usage later